server/models/Request.js

```diff
@@ -2,11 +2,11 @@ const mongoose = require("mongoose");
 
 const requestSchema = new mongoose.Schema({
   userId: {
-    type: Number,
+    type: String,
     required: true,
   },
   sitterId: {
-    type: Number,
+    type: String,
```

Comment:
Hey Emmi, this one is on me but we can actually reference the user here. Could you change this into a reference to the User model 😄